google-cloud-tools-plugin/src/com/google/gct/idea/settings/ImportSettings.java

```diff
@@ -122,6 +124,17 @@ public static void doImport(String path) {
       Messages.showErrorDialog(IdeBundle.message("error.reading.settings.file.2", presentableFileName(saveFile), e1.getMessage()),
                                DIALOG_TITLE);
     }
+    finally {
+      try {
+        if (saveZipFile != null) {
+          saveZipFile.close();
+        }
+      } catch (IOException e1) {
+        Messages.showErrorDialog(
+          "Error closing file " + presentableFileName(saveFile) + ".\\nThere was " + e1.getMessage(),
```

Comment:
I added a message to 'CloudToolsBundle.properties' and now use 'com.google.gct.idea.util.GctBundle'.